docker-compose.yml

```diff
@@ -1,30 +1,34 @@
 version: "3"
 services:
-    posteewebhook:
+    postee:
+        container_name: postee
         build:
             context: .
             dockerfile: Dockerfile
         ports:
             - 8082:8082
         volumes:
             - bolt:/server/database
-            - config:/server
+            - config:/config
         environment:
-            - "POSTEE_CFG=/server/cfg.yaml"
+            - "POSTEE_CFG=/config/cfg.yaml"
```

Comment:
Did you forget to check in the change to move `cfg.yaml` into the `config` directory? I don't see it as part of the PR.

Reply:
Docker-compose use this Dockerfile version: https://github.com/aquasecurity/postee/blob/main/Dockerfile#L18

In this PR is the change to the path /config

Reply:
ah gotcha - I missed that!